card/mod/05_standard/set/type/user.rb

```diff
@@ -51,7 +51,7 @@
   subcards['+*roles'] = { content: Card[:administrator].name }
 
   email, password = subcards.delete('+*account+*email'), subcards.delete('+*account+*password')
-  subcards['+*account'] = { '+*email'=>email, '+*password'=>password }
+  subcards.add_field :account, :subcards=> { "+#{Card[:email].name}"=>email, "+#{Card[:password].name}"=>password }
```

Comment:
Line is too long. [115/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.